recipes/runit_systemd.rb

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright:: Copyright (c) 2015 Opscode, Inc.
+#
+# All Rights Reserved
+#
+
+project_name = node['enterprise']['name'].clone
+project_name.gsub!(/_/, '-') if project_name == 'private_chef'
+unit_name = "#{project_name}-runsvdir-start.service"
```

Comment:
@lamont-granquist I see what you are referring to now.  I thought you were commenting on the entire premise of the project name.  Re _ to -, I believe this is just there because of upgrade problems when we did a project rename: https://github.com/opscode-cookbooks/enterprise-chef-common/commit/fa24dac8e7d7742aa1a3d5859c92bf83108d9342